### PR TITLE
Make it so we can trigger nightly from github actions

### PR DESCRIPTION
**Pull request by LFDanLu**
Closes 

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/WAI/ARIA/apg/)

## 🧢 Your Project:

RSP

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -700,3 +700,11 @@ workflows:
...
       - install
       - publish-nightly
+
+  nightly-manual:
+    when:
+      and:
+        - equal: [ "nightly", << pipeline.parameters.GHA_Action >> ]
+        - equal: [ "workflow_dispatch", << pipeline.parameters.GHA_Event >>]
+    jobs:
+      - install
```
Comment (devongovett) [inline .circleci/config.yml:710]: will need publish-nightly here too right?

Comment (LFDanLu) [reply .circleci/config.yml:710]: yeah, right now im trying to see if I can get the manual trigger button to appear in the UI and test that it fires, omitted nightly for now so I don't accidentally trigger it

Comment (LFDanLu) [reply .circleci/config.yml:710]: Ok apparently the manual trigger button will only appear when the action yaml exists on main. Hopefully we don't run into https://github.com/orgs/community/discussions/25219 after merging

**Review by devongovett — APPROVED** [review]
🤞

**Pull request merged**